src/App.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import './App.scss';
+
+import todos from './api/todos';
+import users from './api/users';
+
+import { TodoList } from './components/TodoList';
+
+const preparedTodos = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId),
+}));
+
+function App() {
+  return (
+    <TodoList todos={preparedTodos} />
```

Comment:
Need to have some container tag for custom component. Also, there should be one `h1` header per page.
```suggestion
    <div>
      <h1>Some title...</h1>
      <TodoList todos={preparedTodos} />
    </div>
```